format_checker/forked-projects.json

```diff
@@ -467,4 +468,4 @@
   "https://github.com/zeroturnaround/gradle-jrebel-plugin": "unforked",
   "https://github.com/zhangxd1989/spring-boot-cloud": "unforked",
   "https://github.com/zhurlik/gradle-jboss-modules-plugin": "unforked"
-}
```

Comment:
What editor/IDE are you using to get this change automatically? At some point we may accept that change to stop dealing with unnecessary changes.

Reply:
I usually use vs code but for this change I directly made in GitHub by editing the file

Reply:
Can you test on another file if the GitHub editor automatically removes or adds newline at the end of the file?

Reply:
I edited another file requirements.txt. I just removed 1 and added 1 in the last line. That line is shown in the diff despite being exactly same.

Reply:
The line is not the same if it's in the diff. Can you search online about reports of GitHub editor adding/removing the newline character? Can that behavior be customized?

Reply:
In "Files changed" tab > settings > I checked the box "Hide whitespaces" and all the unnecessary diff is removed. Only the changed lines are shown.

Reply:
I searched online and in github settings, but did not find any setting to control new lines in the gitghub web editor. But there are options to manage these configurations at IDE level like in VS code. Also, there is an option in GitHub to edit a file in github.dev which opens the repo in an online vs code editor.

Reply:
In this PR, I was not able to reporoduce the exact diff in new branches, I may have inadvertantly changed some space

Reply:
I found https://stackoverflow.com/questions/68088682/how-do-i-stop-githubs-web-editor-from-adding-a-newline-to-the-end-of-file but didn't test anything.

Please remove unnecessary changes so I can accept your PR.

Reply:
Removed unnecessary changes